### PR TITLE
V7 docs

### LINKED ISSUE #7: update docs for upgrading from node 8.9.4 to node 12 to include pm2-windows-service upgrade error mitigations
Manju D J from marlabs reports pm2-windows-service has issues with node 12 on azure. including:

1.) Earlier installed pm2 windows service was pointing to older version of nvm folder  which was not valid.

2.) System variable “PM2_SERVICE_PM2_DIR” was pointing to wrong nvm location.

3.) Inquirer module was not updated to newer version to support latest npm version.

3 is resolved with https://github.com/jon-hall/pm2-windows-service/issues/56
 
Need to update docs.

### LINKED ISSUE #13: Update folder structure
Update the development branch folder structure to more closely match the current main branch for easier deploys.

### LINKED ISSUE #37: Need to update DAT zoom setting
Staging had zoom setting at 1.0 which was causing text templates to produce fonts that were too small for their set size. Mike and I tested it and he found the 1.6 setting most closely matches expected font sizes in PDFs. 

need to update the zoom setting to 1.6 in all environments and restart

### PR DESCRIPTION
**Pull request by tobiasnteireho**
update documentation for DAT. Addresses calibration, azure upgrade issues, and nvm upgrade issues

fix #37 
fix #13 
fix #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,94 +122,87 @@ The recommended process manager is pm2 (http://pm2.keymetrics.io/). Install it w
...
-GUIDES_DIR-  required to establish location of templates
-GUIDES_URL- relative web location of guides
-WKHTMLTOPDF_PATH- path to binary of WKHTMLTOPDF
-
-and these new keys:
+`GUIDES_DIR`-  required to establish location of templates
+`GUIDES_URL`- relative web location of guides
+`WKHTMLTOPDF_PATH`- path to binary of WKHTMLTOPDF
 VIEWER_PATH- path to viewer
 WKHTMLTOPDF_DPI- desired default dpi to render pdfs. CALI reccomends minimum of 300
```
Comment (mitch-mitchel) [inline README.md]: recommends

**Review by mitch-mitchel — CHANGES_REQUESTED** [review]
looks good, just found the one typo

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ While other server environments may work, they have not been tested.  Should you
 
 ## Upgrade notes/summary from node A2JDAT v2
 
-*The folder structure has changed.* It is recommended that your current setup is backed up. This means at a minimum, config.json and ecosystem.config.js. Alternatively you may checkout/unzip this repo into  a new folder.
+**The folder structure has changed.** It is recommended that your current setup is backed up. This means at a minimum, config.json and ecosystem.config.js. Alternatively you may checkout/unzip this repo into  a new folder.
```
Comment (JessicaFrank) [inline README.md]: double space after the word "into"

File: README.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ While other server environments may work, they have not been tested.  Should you
 
 ## Upgrade notes/summary from node A2JDAT v2
 
-The folder structure has changed. It is recommended that your current setup is backed up. This means at a minimum, config.json and ecosystem.config.js. Alternatively you may checkout/unzip this repo into  a new folder.
+*The folder structure has changed.* It is recommended that your current setup is backed up. This means at a minimum, config.json and ecosystem.config.js. Alternatively you may checkout/unzip this repo into  a new folder.
```
Comment (JessicaFrank) [inline README.md]: double space after word "into"

File: README.md (hunk 1 above)
Comment (tobiasnteireho) [reply README.md]: fixed in latest commit

File: README.md (hunk 2 above)
Comment (tobiasnteireho) [reply README.md]: fixed in latest commit

**Review by mitch-mitchel — APPROVED** [review]
👍

**Pull request merged**